### PR TITLE
Appsembler/metalogix/dogwood/feature/taxoman

### PR DESCRIPTION
**Pull request by johnbaldwin**
This is the Taxoman integration to Metalogix dogwood.

It should be straightforward. Comparables are the nxp/release branch and the taxoman-integration branch. 

This code has been tested in devstack, but not on GCP

### CODE REVIEW

**Comment by johnbaldwin** [comment]
[(8) Taxoman: Install on Metalogix production site (Dogwood)](https://trello.com/c/k9qUkpwg/1413-8-taxoman-install-on-metalogix-production-site-dogwood)

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -86,7 +86,7 @@ git+https://github.com/edx/edx-oauth2-provider.git@0.5.8#egg=edx-oauth2-provider
 git+https://github.com/edx/edx-val.git@0.0.8#egg=edxval==0.0.8
 -e git+https://github.com/pmitros/RecommenderXBlock.git@518234bc354edbfc2651b9e534ddb54f96080779#egg=recommender-xblock
 -e git+https://github.com/pmitros/RateXBlock.git@367e19c0f6eac8a5f002fd0f1559555f8e74bfff#egg=rate-xblock
--e git+https://github.com/edx/edx-search.git@release-2015-11-17#egg=edx-search==0.1.1
+-e git+https://github.com/edx/edx-search.git@metalogix/taxoman#egg=edx-search==0.1.1
```
Comment (melvinsoft) [inline requirements/edx/github.txt]: @johnbaldwin Looking at the [edx-search PR](https://github.com/appsembler/edx-search/pull/6), I think the branch should be `metalogix/master`.

Comment (johnbaldwin) [inline requirements/edx/github.txt]: @melvinsoft Ahh, yes, excellent, thank you!

**Comment by johnbaldwin** [comment]
@melvinsoft I updated github.txt. I will test it on staging again. Please review again

**Pull request merged**